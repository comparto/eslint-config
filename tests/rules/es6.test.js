const es6 = require('../../src/rules/es6')

test('es6 rules', () => {
  expect(es6).toMatchSnapshot()
})
