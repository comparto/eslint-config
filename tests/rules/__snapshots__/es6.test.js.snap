// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`es6.js 1`] = `
Object {
  "rules": Object {
    "prefer-arrow-callback": "error",
    "prefer-const": "error",
  },
}
`;
